test/relations.test.js

```diff
@@ -820,12 +820,23 @@ describe('relations', function() {
             done();
           });
         });
-        it('returns empty result when filtering with wrong id key', function(done) {
-          var wrongWhereFilter = {where: {wrongIdKey: samplePatientId}};
-          physician.patients(wrongWhereFilter, function(err, ch) {
+        it('returns patient where name equal to samplePatient name', function(done) {
+          var whereFilter = {where: {name: 'a'}};
+          physician.patients(whereFilter, function(err, ch) {
+            if (err) return done(err);
+            should.exist(ch);
+            ch.should.have.lengthOf(1);
+            ch[0].name.should.eql('a');
+            done();
+          });
+        });
+        it('returns patient where id equal to samplePatientId', function(done) {
```

Comment:
@joostdebruijn This is a duplicate test case, see https://github.com/joostdebruijn/loopback-datasource-juggler/blob/301505fb27acca1fdddc0463bbd26cc847e6029a/test/relations.test.js#L813, we can remove it, see if it fixes the cassandra failure.

Reply:
Ah, I see. It's removed now.